### PR TITLE
Change objective and reference naming

### LINKED ISSUE #37: MANT: change morph data naming
Current proposal:

`objective -> morphed` : data will be manipulated
`reference -> target`: data will stay unchanged.

The `obj-ref` language propagates across the repo (like `xref, yobj` etc), but the convention is rather clear and self-consistent as we always see `xref, yref` remain unchanged in the code.
 
The first order fix could be changing the language in `pdfmorph` function and app since they are user facing.

cc: @sbillinge @pavoljuhas

### LINKED ISSUE #46: Give file names instead of "reference" and "objective" in the plots

### LINKED ISSUE #55: clean up directory structure so tutorial things aren't called tests
remove /tests directory from top level

### PR DESCRIPTION
**Pull request by Sparks29032**
Fixes #37. Fixes #46. Fixes #55.

### CODE REVIEW

File: diffpy/pdfmorph/pdfmorphapp.py [hunk 1]
```diff
@@ -178,6 +178,24 @@ def custom_error(self, msg):
...
+        metavar="MLABEL",
+        dest="mlabel",
+        help="Set label for morphed data to MLABEL on plot. Ignored if using file names as labels.",
+    )
+    group.add_option(
+        '--tlabel',
+        metavar="TLABEL",
+        dest="tlabel",
+        help="Set label for target data to TLABEL on plot. Ignored if using file names as labels.",
+    )
```
Comment (Sparks29032) [inline diffpy/pdfmorph/pdfmorphapp.py:198]: New option to set label names on plot.

File: diffpy/pdfmorph/morphs/morph.py [hunk 2]
```diff
@@ -24,11 +24,11 @@
 
 
 class Morph(object):
-    '''Base class for implementing a morph on an objective given a reference.
+    '''Base class for implementing a morph given a target.
 
     Adapted from diffpy.pdfgetx to include two sets of arrays that get passed
-    through. In most cases, the objective is modified by a morph, but it is
-    acceptable for morph the reference as well, such as to change the range of
+    through. In most cases, only the morph is modified, but it is
```
Comment (sbillinge) [inline diffpy/pdfmorph/morphs/morph.py:30]: Let's remove this sentence.  Changing the range of an array is not a morphing operation.

File: diffpy/pdfmorph/morphs/morphpdftordf.py [hunk 3]
```diff
@@ -24,7 +24,7 @@
 class MorphXtalPDFtoRDF(Morph):
```
Comment (sbillinge) [inline diffpy/pdfmorph/morphs/morphpdftordf.py:24]: this is not a "morph" this is simply a transformation.  Please can we change this language?  I want to merge this so please make an issue to change the language (and name) on this.

Having said that, leave all the changes you have done and I will merge it,  but let's think a bit about the class name and the docstring/documentation.  I should probably understand exactly what is being done here before we finalize this, but stick it on an issue (and add it to the release milestone) so we loop back to it

File: diffpy/pdfmorph/morphs/morphrdftopdf.py [hunk 4]
```diff
@@ -24,7 +24,7 @@
 class MorphXtalRDFtoPDF(Morph):
```
Comment (sbillinge) [inline diffpy/pdfmorph/morphs/morphrdftopdf.py:24]: see above

File: diffpy/pdfmorph/pdfmorphapp.py [hunk 5]
```diff
@@ -178,6 +178,24 @@ def custom_error(self, msg):
         dest="plot",
         help="Do not show the plot.",
     )
+    group.add_option(
+        '--usefilenames',
+        action="store_true",
+        dest="usefilenames",
+        help="Use the file names as labels on plot."
```
Comment (sbillinge) [inline diffpy/pdfmorph/pdfmorphapp.py:185]: isn't this the default behavior?  So we don't need an option for this, just do this if one of the other options is not selected?

File: diffpy/pdfmorph/pdfmorphapp.py (hunk 1 above)
Comment (sbillinge) [reply diffpy/pdfmorph/pdfmorphapp.py:198]: looks good, see comment.   Please put different issues on different PRs

**Review by sbillinge — COMMENTED** [review]
Changes look great, thanks!   

Please see quick inline comments and I can merge.

Please put less on each PR to make it easier to review and merge.  One thing per branch/pr ideally.  If something is  a quick cleaning thing you can add it to another one, but this should have been at least two PRs if not three.

Thanks so much

File: diffpy/pdfmorph/pdfmorphapp.py (hunk 5 above)
Comment (Sparks29032) [reply diffpy/pdfmorph/pdfmorphapp.py:185]: default behavior is to use "target" and "morph" as names

File: diffpy/pdfmorph/morphs/morph.py (hunk 2 above)
Comment (Sparks29032) [reply diffpy/pdfmorph/morphs/morph.py:30]: okie

File: diffpy/pdfmorph/morphs/morphpdftordf.py (hunk 3 above)
Comment (Sparks29032) [reply diffpy/pdfmorph/morphs/morphpdftordf.py:24]: Yep, I'll open a new issue for this (and below)

**Comment by Sparks29032** [comment]
> Changes look great, thanks!
> 
> Please see quick inline comments and I can merge.
> 
> Please put less on each PR to make it easier to review and merge. One thing per branch/pr ideally. If something is a quick cleaning thing you can add it to another one, but this should have been at least two PRs if not three.
> 
> Thanks so much

I will put less in each PR in the future, sorry for bundling them all together this time

File: diffpy/pdfmorph/morphs/morphpdftordf.py (hunk 3 above)
Comment (Sparks29032) [reply diffpy/pdfmorph/morphs/morphpdftordf.py:24]: I am having trouble adding the issue to the milestone

File: diffpy/pdfmorph/pdfmorphapp.py (hunk 5 above)
Comment (sbillinge) [reply diffpy/pdfmorph/pdfmorphapp.py:185]: let's revisit this.  I will merge this and we can have a new quick issue after some discussion what is the best design

**Pull request merged**